varya/classes/class-varya-custom-colors.php

```diff
@@ -262,6 +267,31 @@ function varya_editor_custom_color_variables() {
 		}
 	}
 
+	/**
+	 * Customizer contrast validation warnings.
+	 * 
+	 * @author Per Soderlind
+	 * @link https://github.com/soderlind/2016-customizer-demo
+	 */
+	function on_customize_controls_enqueue_scripts() {
+		$handle = 'wcag-validate-customizer-color-contrast';
+		$src    = get_template_directory_uri() . '/assets/js/customizer-validate-wcag-color-contrast.js';
+		$deps 	= [ 'customize-controls' ];
+
+		$exports = [
+			'validate_color_contrast' => [
+				// key = current color control , values = array with color controls to check color contrast against
+				'varya_--global--color-primary' => [ "varya_--global--color-background", "varya_--global--color-background-light" ],
+				'varya_--global--color-foreground' => [ "varya_--global--color-background", "varya_--global--color-background-light" ],
+				'varya_--global--color-background' => [ "varya_--global--color-primary", "varya_--global--color-foreground" ],
+				'varya_--global--color-background-light' => [ "varya_--global--color-primary", "varya_--global--color-foreground" ],
+			],
```

Comment:
Also noting that we recently realized that `varya_--global--color-background-light` isn’t very useful conceptually. It will likely become `varya_--global--color-tertiary` or `varya_--global--color-accent` so that theme designers can choose to use an extra color in child themes similar to a primary or secondary color. 

This means the colors won’t need to be evaluated against `varya_--global--color-background-light` at all. Instead, I’m thinking we’ll only need to evaluate the colors against the `varya_--global--color-background` color. We can do this in a separate PR after this is merged.